src/server/rdb_load.cc

```diff
@@ -2280,6 +2280,17 @@ error_code RdbLoader::Load(io::Source* src) {
       continue;
     }
 
+    if (type == RDB_OPCODE_SLOT_INFO) {
+      [[maybe_unused]] uint64_t slot_id;
+      SET_OR_RETURN(LoadLen(nullptr), slot_id);
+      [[maybe_unused]] uint64_t slot_size;
+      SET_OR_RETURN(LoadLen(nullptr), slot_size);
+      [[maybe_unused]] uint64_t expires_slot_size;
+      SET_OR_RETURN(LoadLen(nullptr), expires_slot_size);
+      LOG(WARNING) << "Loaded and ignored RDB_OPCODE_SLOT_INFO";
```

Comment:
I believe we dont need to print this warning

Reply:
other than that lgtm

Reply:
sounds good